backends/dpdk/dpdkHelpers.cpp

```diff
@@ -632,7 +632,11 @@ bool ConvertStatementToDpdk::preorder(const IR::AssignmentStatement *a) {
         }
     } else if (right->is<IR::PathExpression>() || right->is<IR::Member>() ||
                right->is<IR::BoolLiteral>() || right->is<IR::Constant>()) {
-        i = new IR::DpdkMovStatement(a->left, a->right);
+        if (right->is<IR::Constant>() && right->type->to<IR::Type_Bits>()->width_bits() == 128) {
```

Comment:
What happen for value greater than 64 bits and less than 128 bits

Reply:
Are you throwing an error for these cases? If so, where?

Reply:
It will be thrown in dpdkArch.h itself, These cases will not reach this point of code.